doc/CustomRules.md

```diff
@@ -6,6 +6,12 @@ Custom rules can do everything the built-in rules can and are defined inline or
 ([keyword `markdownlint-rule` on npm](https://www.npmjs.com/search?q=keywords:markdownlint-rule)).
 Custom rules can be disabled, enabled, and customized using the same syntax as built-in rules.
 
+## Implementing simple rules
+
+For simple requirements, like disallowing certain characters or patterns, community developed
+[markdownlint-rule-search-replace](https://www.npmjs.com/package/markdownlint-rule-search-replace) plug-in can be used.
+With it a rule can be created by simply adding a configuration.
```

Comment:
Instead, maybe: This plug-in allows anyone to create a set of simple text-replacement rules in JSON without needing to write any code.